README.md

```diff
@@ -14,10 +14,10 @@ Just a repo for me to keep my custom badges around
 
 To link to [slides.com](https://slides.com/) slides
 
-<img alt="slides" src="badges/slides.png" height="20" />
+[![slides-badge](badges/slides.svg)](http://example.com/your-slides)
 
 ```
-[<img alt="slides" src="https://cdn.rawgit.com/kentcdodds/custom-badges/VERSION/badges/slides.png" height="20" />](http://example.com/your-slides)
+[![slides-badge](badges/slides.svg)](http://example.com/your-slides)
```

Comment:
For this one, it's actually an example to people of how to use the badge, so it's important that it's the full URL, like this:

```
[![slides-badge](https://cdn.rawgit.com/kentcdodds/custom-badges/VERSION/badges/slides.svg)](http://example.com/your-slides)
```